vendor/engines/secure_rooms/app/models/secure_room_price_policy.rb

```diff
@@ -2,4 +2,22 @@ class SecureRoomPricePolicy < PricePolicy
 
   include PricePolicies::Usage
 
+  def estimate_cost_and_subsidy_from_order_detail(order_detail)
+    calculate_cost_and_subsidy_from_order_detail(order_detail)
+  end
+
+  def calculate_cost_and_subsidy_from_order_detail(order_detail)
+    entry_at = order_detail.occupancy.entry_at
```

Comment:
I'm assuming at this point that `occupancy` exists. I'd like to keep it that way for now, but if it turns out to not be the case, we'll learn a little bit of how the code works, and we can drop in a `try`.